dev/com.ibm.ws.rest.handler.config_fat/fat/src/com/ibm/ws/rest/handler/config/fat/ConfigOpenApiSchemaTest.java

```diff
@@ -44,6 +48,12 @@
     @Server("com.ibm.ws.rest.handler.config.openapi.fat")
     public static LibertyServer server;
 
+    @ClassRule
+    public static RepeatTests r = MicroProfileActions.repeat(null, TestMode.FULL,
```

Comment:
Why pass null here rather than the server?

Reply:
That's a mistake, I've copied it from the other test where it's at the suite level.